legend-pure-runtime-java-engine-compiled/src/main/java/org/finos/legend/pure/runtime/java/compiled/generation/processors/support/CompiledSupport.java

```diff
@@ -386,11 +386,31 @@ public static int safeHashCode(Object object)
         return object.hashCode();
     }
 
+    public static <T> T toOneWithAssert(T object, String message, SourceInformation sourceInformation)
+    {
+        return toOne(object, message, sourceInformation);
+    }
+
+    public static <T> T toOneWithAssert(RichIterable<? extends T> objects, String message, SourceInformation sourceInformation)
+    {
+        return toOne(objects, message, sourceInformation);
+    }
+
```

Comment:
I don't think these functions serve much purpose. Just make the toOne functions with the message parameter public. The same comment applies to the toOneManyWithAssert functions.

Reply:
Makes sense. Thank you for reviewing!